src/benchmark/tpch_benchmark.cpp

```diff
@@ -0,0 +1,355 @@
+#include <chrono>
+#include <fstream>
+#include <iostream>
+#include <random>
+#include <string>
+
+#include "cxxopts.hpp"
+#include "json.hpp"
+#include "sql/sql_pipeline.hpp"
+#include "tpch/tpch_db_generator.hpp"
+#include "tpch/tpch_queries.hpp"
+#include "SQLParser.h"
+#include "SQLParserResult.h"
+
+/**
+ * This benchmark measures Hyrise's performance executing the TPC-H queries, it doesn't (yet) support running the TPC-H
+ * benchmark as it is specified.
```

Comment:
The comment should include what is missing.

Reply:
added a short explanation and referred to the tpc